Packs/ExodusIntelligence/README.md

```diff
@@ -0,0 +1,9 @@
+**Exodus Intelligence’s unmatched vulnerability research capabilities and products provide unique context around your enterprise infrastructure allowing you to better prioritize protection and mitigation efforts. This rich context in conjunction with Cortex XSOAR enables many value-add use cases for alerting and contextual enrichment that would otherwise not be possible.**
```

Comment:
Please provide example for the commands and generate the readme again, as I believe it was missed.